src/main/java/techcourse/myblog/domain/article/Article.java

```diff
@@ -19,9 +20,19 @@
     private Long id;
 
     private String title;
```

Comment:
`@Column`을 사용하여 데이터의 속성을 표현해 보면 어떨까요?

```suggestion
    @Column(nullable = false)
    private String title;
```

Reply:
`@Column` 애너테이션은, 
DB에 매핑될 컬럼에 대해 명확히 메타정보를 남기는 것에 의미가 있습니다.
`@Column` 애너테이션을 적극적으로 사용했습니다.